### DISCUSSION TITLE
blog/rust-first-impressions-through-the-eyes-of-a-jspython-coder

### DISCUSSION THREAD

**Discussion by giscus** [masnormen/site#3] [Comments]
# blog/rust-first-impressions-through-the-eyes-of-a-jspython-coder

Nourman Hajar

http://nourman.com/post/rust-first-impressions-through-the-eyes-of-a-jspython-coder

**Comment by masnormen** [comment]
Testing komentar 😄

**Reply by masnormen** [reply]
test
